swagger.yml

```diff
@@ -25,16 +25,16 @@ paths:
       summary: "Create or update a claim."
       operationId: "createOrUpdateClaimUsingPOST"
       consumes:
-        - "application/json"
```

Comment:
I have no idea why this formatting happened. Did a `./gradew clean build` multiple times. Can someone else do a build and see what they get.

Reply:
Yep, I just did a build on master and the exact same changes are present for me too, odd.